README.md

```diff
@@ -115,3 +115,78 @@ if (handleStartupEvent()) {
   return;
 }
 ```
+
+## Checking for updates, downloading and installing
```

Comment:
That code predates it being supported in Electron.

Atom only recently upgraded to the minimum Electron version supporting this and they haven't removed that unneeded code yet.